<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>40</y>
      <width>332</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Meniu Principal</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clienti">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>120</y>
      <width>252</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clienti</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_tranzactii">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>180</y>
      <width>252</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tranzactii</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_arme">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>240</y>
      <width>252</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Arme</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_tipuri">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>300</y>
      <width>252</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tipuri Arme</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_conditii">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>360</y>
      <width>252</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Conditii Arme</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_munitie">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>420</y>
      <width>252</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Munitie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_poligon">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>480</y>
      <width>252</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Poligon</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
